config/checkstyle_sevntu_checks.xml

```diff
@@ -0,0 +1,227 @@
+<?xml version="1.0" ?>
+
+<!DOCTYPE module PUBLIC
+        "-//Checkstyle//DTD Checkstyle Configuration 1.2//EN"
+        "https://checkstyle.org/dtds/configuration_1_2.dtd">
+
+<module name="Checker">
+  <!-- All checks are from https://github.com/sevntu-checkstyle/sevntu.checkstyle project -->
+
+  <!-- Filters -->
+  <module name="SuppressionFilter">
+    <property name="file" value="config/sevntu_suppressions.xml"/>
+  </module>
+
+  <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchFilter">
+    <property name="file" value="show.patch"/>
+    <property name="strategy" value="patchedline" />
+  </module>
+
+  <module name="TreeWalker">
+    <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionJavaPatchFilter">
+      <property name="file" value="show.patch"/>
```

Comment:
`target/show.patch` , the same for above

Reply:
done